src/libraries/System.Private.CoreLib/src/CompatibilitySuppressions.xml

```diff
@@ -17,4 +17,16 @@
     <DiagnosticId>CP0002</DiagnosticId>
     <Target>F:System.Resources.ResourceSet.Reader</Target>
   </Suppression>
-</Suppressions>
\ No newline at end of file
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.String.Trim(System.ReadOnlySpan{System.Char})</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.String.TrimEnd(System.ReadOnlySpan{System.Char})</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.String.TrimStart(System.ReadOnlySpan{System.Char})</Target>
+  </Suppression>
+</Suppressions>
```

Comment:
If we haven't already, I assume our intent is to follow-up for net10.0 with a change to remove these and delete the source impl?

Reply:
The current intent is both main and ~9.0.1, just letting prebuilt things get through a cycle of the method "not being there" without the MissingMethodException.